File: app/src/main/java/com/assodikyhilmy/myfirstandroidgame/GameActivity.java
package com.assodikyhilmy.myfirstandroidgame;

import android.content.pm.ActivityInfo;
import android.graphics.Point;
import android.os.Bundle;
import android.support.v7.app.AppCompatActivity;
import android.view.Display;

/**
 * Created by lenovo on 12/08/2017.
 */

public class GameActivity extends AppCompatActivity {

    //declaring gameview
    private GameView gameView;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);

        //setting the orientation to landscape
        setRequestedOrientation(ActivityInfo.SCREEN_ORIENTATION_LANDSCAPE);

        //Getting display object
        Display display = getWindowManager().getDefaultDisplay();

        //Getting the screen resolution into point object
        Point size = new Point();
        display.getSize(size);

        //Initializing game view object
        //this time we are also passing the screen size to the GameView constructor
        gameView = new GameView(this, size.x, size.y);
        gameView.setKeepScreenOn(true);

        //adding it to contentview
        setContentView(gameView);
    }

    //pausing the game when activity is paused
    @Override
    protected void onPause() {
        super.onPause();
        gameView.pause();
    }

    //running the game when activity is resumed
    @Override
    protected void onResume() {
        super.onResume();
        gameView.resume();
    }
}

File: app/src/main/java/com/assodikyhilmy/bantugatot/activities/GameActivity.kt
package com.assodikyhilmy.bantugatot.activities

import android.content.pm.ActivityInfo
import android.graphics.Point
import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity

/**
 * Created by lenovo on 12/08/2017.
 */
class GameActivity : AppCompatActivity() {
    //declaring gameview
    private var gameView: GameView? = null
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)

        //setting the orientation to landscape
        requestedOrientation = ActivityInfo.SCREEN_ORIENTATION_LANDSCAPE

        //Getting display object
        val display = windowManager.defaultDisplay

        //Getting the screen resolution into point object
        val size = Point()
        display.getSize(size)

        //Initializing game view object
        //this time we are also passing the screen size to the GameView constructor
        gameView = GameView(this, size.x, size.y)
        gameView!!.keepScreenOn = true

        //adding it to contentview
        setContentView(gameView)
    }

    //pausing the game when activity is paused
    override fun onPause() {
        super.onPause()
        gameView!!.pause()
    }

    //running the game when activity is resumed
    override fun onResume() {
        super.onResume()
        gameView!!.resume()
    }
}